### DISCUSSION TITLE
How to add new task in Loco?

### DISCUSSION THREAD

**Discussion by sidoqi** [loco-rs/loco#267] [Q&A] [answered]
I'm trying to add new task in Loco following steps in [Loco Guide](https://loco.rs/docs/getting-started/guide/).

However, I can't run the task since it's not found.

My question is: Do we need to register the task somewhere? 

What i've done so far:
- Add new task script under `tasks` folder.
- Register it in `tasks/mod.rs`.

Am I missing something. Thanks!

**Comment by kaplanelad** [comment, marked as answer]
Hi @jurukode, we have a doc on creating a task [here](https://loco.rs/docs/the-app/task/).
You have two options:
1. add it by loco generator with the following command: `cargo loco generate task`
2. add it manually. Read the docs [here](https://loco.rs/docs/the-app/task/#creating-a-task-manually)

**Reply by sidoqi** [reply]
Thank you sir!
